playbooks/cluster/openshift/config.yml

```diff
@@ -2,18 +2,57 @@
 - hosts: all
   remote_user: root
   pre_tasks:
+    - name: include_vars
+      include_vars:
+        file: "{{ playbook_dir }}/vars/{{ openshift_ver }}.yml"
+
+    - name: set SELinux to permissive mode
+      command: setenforce 1
+
+    - name: set SELinux to permissive mode under configuration file
+      selinux:
+        policy: targeted
+        state: enforcing
+
+    - name: stop and disable firewalld
+      register: result
+      service:
+        state: stopped
+        enabled: no
+        name: firewalld
+      failed_when: "result|failed and not 'Could not find the requested service' in result.msg"
+
+    - name: install epel repository
+      yum:
+        name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+        state: present
+
     - name: Install openshift_facts requirements
       yum:
         name: "{{ item }}"
       with_items:
         - python-yaml
         - python-ipaddress
+        - wget
+        - git
+        - net-tools
+        - bind-utils
+        - iptables-services
+        - bridge-utils
+        - bash-completion
+        - kexec-tools
+        - sos
+        - psacct
+        - docker
+
```

Comment:
I deployed a cluster using the standard CentOS image from https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2

To get openshift-ansible to run, I had to install:
yum-utils ansible wget git net-tools bind-utils iptables-services bridge-utils bash-completion kexec-tools sos psacct docker sshpass NetworkManager centos-release-openshift-origin.noarch ceph-common docker centos-release-openshift iptables-utils NetworkManager atomic-openshift-utils

And ensure NetworkManager is started and enabled

Reply:
Prerequisites that don't appear here (such as NetworkManager, and centos-release-openshift) are being cared as part of the openshift-ansible modules.

Reply:
Didn't work for me until I manually installed them before running deploy_cluster.yml